desktop/config.coffee

```diff
@@ -6,98 +6,101 @@
   MAX_POLLS_FOR_MAX_BIDS: 20
+  MAX_SOCKETS: -1
+  METAPHYSICS_ENDPOINT: null
+  MOBILE_MEDIA_QUERY: "only screen and (max-width: 640px)"
+  MOBILE_URL: 'https://m.artsy.net'
+  NODE_ENV: 'development'
+  OPENREDIS_URL: null
   PARSELY_KEY: 'artsy.net'
   PARSELY_SECRET: null
-  PREDICTION_URL: 'https://live.artsy.net'
-  SEGMENT_WRITE_KEY_MICROGRAVITY: null
-  GEODATA_URL: 'http://artsy-geodata.s3-website-us-east-1.amazonaws.com'
-  EMAIL_SIGNUP_IMAGES_ID: null
-  IP_BLACKLIST: ''
-  CRITEO_AUCTIONS_ACCOUNT_NUMBER: '28539'
-  CRITEO_ARTWORKS_ACCOUNT_NUMBER: '35250'
-  APPLY_URL: 'http://apply.artsy.net'
   PC_ARTSY_CHANNEL: '5759e508b5989e6f98f77999'
   PC_AUCTION_CHANNEL: '5759e4d7b5989e6f98f77997'
-  GALLERY_INSIGHTS_CHANNEL: '5759e4a6b5989e6f98f77995'
-  EOY_2016: '5829db77b5989e6f98f779a5'
-  EDITORIAL_CTA_BANNER_IMG: 'http://files.artsy.net/images/iphone_email.png'
-  GALLERY_PARTNER_UPDATES_CHANNEL: '5762d454b5989e6f98f7799a'
-  TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$'
-  MARKETING_SIGNUP_MODALS: '[{"slug":"ca1","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-interior-a-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca2","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-interior-a-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca3","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-interior-b-credited.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca4","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-interior-b-credited.jpg","photoCredit":"Photo by Emily Johnson"}]'
-  EOY_2016_ARTICLE: null
+  PORT: 3004
+  POSITRON_URL: 'http://localhost:3005'
+  PREDICTION_URL: 'https://live.artsy.net'
   RAYGUN_KEY: null
+  REFLECTION_URL: 'http://artsy-reflection.s3-website-us-east-1.amazonaws.com/__reflection/forceartsynet'
+  REQUEST_EXPIRE_MS: 60000 # 1 minute
+  REQUEST_LIMIT: 120
+  S3_KEY: null
+  S3_SECRET: null
   S3_BUCKET: null
+  SAILTHRU_AUCTION_NOTIFICATION_LIST: 'Auction Notifications'
+  SAILTHRU_KEY: null
+  SAILTHRU_MASTER_LIST: 'Master List'
+  SAILTHRU_SECRET: null
+  SECURE_IMAGES_URL: null
   SEGMENT_AMP_WRITE_KEY: null
-  WHITELISTED_VANITY_ASSETS: ''
+  SEGMENT_WRITE_KEY: null
+  SEGMENT_WRITE_KEY_MICROGRAVITY: null
+  SESSION_COOKIE_KEY: 'force.sess'
+  SESSION_COOKIE_MAX_AGE: 31536000000
+  SESSION_SECRET: 'change-me'
+  SENTRY_PRIVATE_DSN_STAGING: 'https://7470396d71fe45278fa624aae59505c8:b697285749b5484c81bf730b9e0dd485@sentry.io/159064'
+  SENTRY_PRIVATE_DSN_PRODUCTION: 'https://814ffa3a48684455a9fc500e18225403:0d14e90f25ec4bac96d475582504ecca@sentry.io/28316',
```

Comment:
Be sure to exclude any private keys from being checked in. These config files are simply used to provide defaults for the convenience of development—in production nearly all of them will be overwritten by environment variables. Let's remove these from the config files and make sure we start a new Sentry account/API keys when this goes live.